ros-one.repos

```diff
@@ -341,28 +349,32 @@ repositories:
     version: master
   jsk_common:
     type: git
-    url: https://github.com/ros-o/jsk_common.git
-    version: obese-devel
+    url: https://github.com/mqcmd196/jsk_common.git
+    version: techfak-ros-o
   jsk_common_msgs:
     type: git
     url: https://github.com/jsk-ros-pkg/jsk_common_msgs.git
     version: master
-  jsk_recognition:
-    type: git
-    url: https://github.com/jsk-ros-pkg/jsk_recognition.git
-    version: master
+  jsk_model_tools:
+    type: git
+    url: https://github.com/mqcmd196/jsk_model_tools.git
+    version: techfak-ros-o
+  # jsk_pr2eus:
+  #   type: git
+  #   url: https://github.com/jsk-ros-pkg/jsk_pr2eus.git
+  #   version: master
+  # jsk_recognition:
+  #   type: git
+  #   url: https://github.com/jsk-ros-pkg/jsk_recognition.git
+  #   version: master
   jsk_roseus:
     type: git
     url: https://github.com/jsk-ros-pkg/jsk_roseus.git
     version: master
-  jsk_visualization:
-    type: git
-    url: https://github.com/ros-o/jsk_visualization.git
-    version: obese-devel
-  jskeus:
-    type: git
-    url: https://github.com/tork-a/jskeus-release.git
-    version: release/noetic/jskeus
+  # jsk_visualization:
+  #   type: git
+  #   url: https://github.com/jsk-ros-pkg/jsk_visualization.git
+  #   version: master
```

Comment:
and this?

Reply:
This package requires jsk_recognition_utils in jsk_recognition.

Reply:
https://github.com/ubi-agni/ros-builder-action/pull/36/commits/d8f5881d0c1b958c4fc9b9a9677baa2e1e38faa6